prometheus_client/utils.py

```diff
@@ -18,6 +18,6 @@ def floatToGoString(d):
         # Go switches to exponents sooner than Python.
         # We only need to care about positive values for le/quantile.
         if d > 0 and dot > 6:
-            mantissa = '{0}.{1}{2}'.format(s[0], s[1:dot], s[dot+1:]).rstrip('0.')
+            mantissa = '{0}.{1}{2}'.format(s[0], s[1:dot], s[dot+1:16]).rstrip('0.')
```

Comment:
I'm not sure this is correct, as it's the underlying algorithm that's not right.

Reply:
Are you saying that there's no easy way to preserve compatibility with Python 2.6?

Reply:
Possibly not, we'd need our own floating point renderer for it.

May I ask why you're still on 2.6?

Reply:
For my sins :-P

I'll close the PR then, I'm good with using 0.4.2 until we can upgrade to Python 2.7.